mrrc/__init__.py

```diff
@@ -1,5 +1,11 @@
 from .command import init,upload,delete,gen,ls,cli
+from .mrrc_logger import set_logging
+import logging
 
+# override this however you want
+set_logging(level=logging.INFO)
```

Comment:
why are we calling se_logging here and getLogger in the config.py?

Reply:
This is a universal control to init logging at info level as default. We will have some type of --debug or --verbose flag later in cli to switch to debug level.